src/psyclone/transformations.py

```diff
@@ -2432,8 +2463,8 @@ def apply(self, nodes):
                                       "the loop(s) to which it applies!")
         node_position = node_list[0].position
 
-        # Do the checks in the base class
-        super(ProfileRegionTrans, self)._validate(node_list)
+        # Perform validation checks
+        self._validate(node_list)
```

Comment:
Yes, that was bad (I can write this since I did it ;) )

Reply:
I've only just worked out what you mean :-) I think I'd have made the same mistake!